opencti-platform/opencti-front/src/private/components/cases/feedbacks/FeedbackCreation.tsx

```diff
@@ -73,14 +78,18 @@ const feedbackMutation = graphql`
 
 const caseValidation = () => Yup.object().shape({
   description: Yup.string().nullable(),
+  confidence: Yup.number(),
   rating: Yup.number(),
 });
 
 interface FormikCaseAddInput {
-  description: string;
-  rating: number;
-  objects: { value: string }[];
-  file: File | undefined;
+  description: string
+  confidence: number
+  rating: number
+  objects: { value: string }[]
+  file: File | undefined
+  objectLabel: Option[]
+  externalReferences: { value: string }[]
```

Comment:
External refs not interesting for feedbacks

Reply:
Fixed